.gitattributes

```diff
@@ -1,7 +1,7 @@
 # This line does two things:
-# - enable git's CRLF -> LF modification
+# - forces git to use LF
 # - tell git to guess whether files are text or binary
-*                       text=auto
+*                       text=auto eol=lf
```

Comment:
Why was it necessary?

Reply:
I was fighting with the CI about one of the patches not matching the one I generated, turns out it was because the CI was using `\n` while my machine was using `\r\n`